pwndbg/config.py

```diff
@@ -62,10 +62,13 @@ def __init__(self, name, default, docstring):
                                         gdb.COMMAND_SUPPORT,
                                         getParam(default))
         self.value = default
+        self.trigger = trigger
 
         setattr(module, self.name, self)
 
     def get_set_string(self):
+        if self.trigger:
```

Comment:
I understand the need for "setting changed, trigger function" and this may be useful in the future.

However, Pwndbg currently makes extensive use of annotations, which make it easy to see when a given function is invoked.

I'd much prefer annotation style to this style.  To change to annotation style, one simply needs to implement `__call__` to return a function wrapper, and that should work.

An example that's relevant may be:

```py
@pwndbg.color.hexdump_special
def reload_hexdump_colors():
    ...
```

Or something like that.  This would allow easy registration of *mutliple* functions for a given `Parameter`, and allow easy grepping and see when a given thing is triggered.